docker-compose.yaml

```diff
@@ -141,16 +142,34 @@ services:
     depends_on:
       - keycloak
 
-  keycloak: # https://hub.docker.com/r/jboss/keycloak
+  postgres: # https://hub.docker.com/_/postgres
+    container_name: keycloak-postgres
```

Comment:
can we just call it postgres, or do you think the keycloak part is important to the user?

Reply:
`postgres` is fine, I think this is just a remnant from my local machine testing :)